base/src/test/java/uk/ac/ebi/atlas/profiles/MinMaxProfileRankingTest.java

```diff
@@ -41,7 +43,7 @@ private BaselineProfile randomProfile(double min, double max, String id, List<As
 
         BaselineProfile result = new BaselineProfile(id, id);
         for (AssayGroup factorName: allFactors) {
-            if (rng.nextBoolean() || includeAll) {
+            if (includeAll || rng.nextBoolean() ) {
```

Comment:
Just a curiosity, if you don’t want to declare a `Random` object, etc. you can do this `ThreadLocalRandom.current().nextBoolean()`, `ThreadLocalRandom.current().nextDouble(origin, bound)`, etc.